pages/common/exiftool.md

```diff
@@ -5,20 +5,20 @@
 
 - Remove all EXIF metadata from the given files:
 
-`exiftool -All= {{file}}`
+`exiftool -All= {{file1 file2 ...}}`
 
-- Increase time photo taken by 1 hour in directory:
+- Move the shooting date of all photos in a directory 1 hour forward:
```

Comment:
shooting date sounds a bit awkward. Do you have a better option ?

Reply:
Not really, I was wondering about it myself :\ maybe "move the date at which all photos in a directory were taken ..."? Sounds awkward as well to me. Suggestions welcome!

Reply:
Sounds better than "shooting date"

Reply:
I was trying to find a shorter form though, because the following example seems too long/heavy IMHO:

    - Move the date at which all JPEG photos in the current directory were taken 1 day and 2 hours backward:

Reply:
When it comes to choosing between shorter or clearer, we always choose clearer. So I think it's fine for now.

Reply:
Alright, applied :)